src/components/transport_manager/src/transport_adapter/transport_adapter_impl.cc

```diff
@@ -46,12 +46,11 @@ namespace transport_adapter {
 CREATE_LOGGERPTR_GLOBAL(logger_, "TransportManager")
 namespace {
 DeviceTypes devicesType = {
-    std::make_pair(AOA, std::string("USB_AOA")),
-    std::make_pair(PASA_AOA, std::string("USB_AOA")),
-    std::make_pair(MME, std::string("USB_IOS")),
```

Comment:
@JackLivio According to the agreement we have to use `DEPRECATED` for interfaces only. And DeviceType is just a type(of course it can be part of interface like any other type like `int`, or `std::vector`, but it is not an interface)

Reply:
The reason I ask for this is that integrators might have proprietary transport adapters that use these enums when returning GetDeviceType(). For example integrators that wrote their own IAP transport adapters might be returning "MME" for their IAP::GetDeviceType method.

If we can't find a way to deprecate these enum values using the DEPRECATE keyword, then i suggest we leave the old values in addition to adding the new values and in the comments above the enum declaration / usage we note that PASA_AOA, PASA_BLUETOOTH, and MME are being deprecated.

https://github.com/smartdevicelink/sdl_core/pull/1927/files#diff-b2bb75ea564a7cfc388b4f3cc9980f7f



```
//@deprecated PASA_AOA, PASA_BLUETOOTH, MME
enum DeviceType { AOA, PASA_AOA, BLUETOOTH, PASA_BLUETOOTH, MME, IOS_BT, IOS_USB, TCP, UNKNOWN }; 
```

Reply:
@JackLivio Fixed in 1d710a202cfed11af514ad375019ca284afd9001